components/tag-list/tag-list-item-mixin.js

```diff
@@ -17,30 +19,26 @@ export const TagListItemMixin = superclass => class extends superclass {
 		return [labelStyles, css`
 			:host {
 				display: grid;
-				outline: none;
+				max-width: 100%;
 			}
 			:host([hidden]) {
 				display: none;
 			}
-			.tag-list-item-content {
-				height: 1rem;
-				margin: auto;
-				min-width: 0;
-				overflow: hidden;
-				text-overflow: ellipsis;
-				white-space: nowrap;
-			}
 			.tag-list-item-container {
 				background-color: var(--d2l-color-regolith);
 				border-radius: 6px;
 				box-shadow: inset 0 0 0 1px var(--d2l-color-gypsum), 0 2px 4px rgba(0, 0, 0, 0.03);
 				box-sizing: border-box;
 				color: var(--d2l-color-ferrite);
 				cursor: pointer;
-				display: flex;
+				max-width: 320px;
 				min-width: 0;
+				outline: none;
+				overflow: hidden;
 				padding: 0.25rem 0.6rem;
+				text-overflow: ellipsis;
 				transition: background-color 0.2s ease-out, box-shadow 0.2s ease-out;
+				white-space: nowrap;
 			}
 			:host(:hover) .tag-list-item-container,
 			:host(:focus) .tag-list-item-container {
```

Comment:
I'm not sure what the focus styles are supposed to look like... currently it looks like:
![image](https://user-images.githubusercontent.com/9042472/162440729-cf15864d-7364-45c9-9c27-3ce06806f5ce.png)
... which doesn't seem to match the styling we've used in several other places.
![image](https://user-images.githubusercontent.com/9042472/162440836-9c7270cc-16a4-46bf-ab4f-e4b1941e7101.png)
I imagine we want tags to look a little different from buttons though...?  Still seems a little off to me... not sure if Design is still iterating. 🤷‍♂️

Reply:
Oh interesting - I actually am not seeing the extra outline when I run it so I'll dig into why that might be happening.